functions/supplier/sftp-supplier/pom.xml

```diff
@@ -30,6 +30,11 @@
             <artifactId>spring-integration-sftp</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.cloud.fn</groupId>
+            <artifactId>config-common</artifactId>
```

Comment:
I guess we need this for `SpelExpressionConverterConfiguration`.
Can  we consider apply it then on the `SftpSupplierConfiguration`, but not only in the test?
I mean without such an import the final function app is not going to work just because you have already provided those `Expression` configuration properties. So, better to keep it working.